Orange/widgets/utils/domaineditor.py

```diff
@@ -322,6 +322,9 @@ def numbers_are_round(var, col_data):
                                 str(int(x)) if round_numbers else
                                 orig_var.repr_val(x)
                                 for x in self._iter_vals(col_data)]
+                elif type(orig_var) == TimeVariable:
+                    col_data = [orig_var.repr_val(x) if not np.isnan(x) else ""
+                                for x in self._iter_vals(col_data)]
```

Comment:
This looks to be the same as in the above `if type(orig_var) == DiscreteVariable`,
we could change the first check to match both types, e.g. `if type(orig_var) in (DiscreteVariable, TimeVariable)`,
so we can avoid copy/pasting code.